honeybee_vtk/model.py

```diff
@@ -187,6 +198,38 @@ def _load_cameras(self, model: HBModel) -> None:
                 'views not found in HBJSON.'
             )
 
+    def _load_data(self, config: dict) -> None:
+        """Mount data on model from config json.
+
+        This method is only used in cli.
```

Comment:
I think this can be a useful method to use in general for adding data to a model.

Reply:
Complied.